int32 id
geometry_msgs/Pose pose
float64 velocity_x
float64 velocity_y
